test/test_sedcli_basic.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (C) 2018-2019 Intel Corporation
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+"""
+Tests end-user scenarios for sedcli binary.
+This suite needs to be run against Python 3.x and the sedcli binary must be
+in your $PATH.
+"""
+
+import os
+import sys
+import unittest
+import subprocess
+import random
+import string
+
+class TestSedcliBasic(unittest.TestCase):
+
+	NVME_DEV_PATH = '/dev/nvme0n1'
+
+	def __init__(self, *args, **kwargs):
+		super(TestSedcliBasic, self).__init__(*args, **kwargs)
+
+		letters = string.ascii_lowercase
+		self.nvme_sid_key =  ''.join(random.choice(letters) for i in range(32)).encode('ascii')
+
+	def test_initial_setup(self):
+		"""
+		Tests for: sedcli intial provision of drive (that is take ownerhisp,
+		activate Locking SP and setup global locking range)
+		"""
+
+		print("Key used to initial provision drive:" + str(self.nvme_sid_key))
+
+		pipe = subprocess.Popen(["sedcli", "--ownership", "--device", self.NVME_DEV_PATH], stdout = subprocess.PIPE, stdin = subprocess.PIPE, stderr = subprocess.PIPE)
+		pipe.stdin.write(self.nvme_sid_key + b'\n' + self.nvme_sid_key + b'\n')
+		outs, errs = pipe.communicate()
+		pipe.stdin.close()

```

Comment:
Utility function can be implemented in order to execute commands and get result. This will make the code more readable